finality-aleph/src/party/manager/mod.rs

```diff
@@ -345,8 +345,14 @@ where
         {
             #[cfg(feature = "only_legacy")]
             _ if self.only_legacy() => self.legacy_subtasks(params),
-            Ok(version) if version == CURRENT_VERSION => self.current_subtasks(params),
-            Ok(version) if version == LEGACY_VERSION => self.legacy_subtasks(params),
+            Ok(version) if version == CURRENT_VERSION => {
+                info!(target: "aleph-party", "Running session with AlephBFT version {}, which is current.", version);
+                self.current_subtasks(params)
+            }
+            Ok(version) if version == LEGACY_VERSION => {
+                info!(target: "aleph-party", "Running session with AlephBFT version {}, which is legacy.", version);
```

Comment:
add also to arm `_ if self.only_legacy() ...` :)

Reply:
Added.